SpecTools/Classes/Find/Find+UIViewController.swift

```diff
@@ -21,22 +21,18 @@ extension SpecFind where T: UIViewController {
     }
     
     public func has(childViewController viewController: UIViewController) -> Bool {
-        // TODO: Check if views are added
-        // TODO: Check if childViewControllers contain viewController
-        return false
+        return element.childViewControllers.contains(viewController)
```

Comment:
we still need to check if `element.view.subviews` contains (maybe even recursively) `viewController.view`

Reply:
I guess the spec traverser could do with a recursive method contains(view) with === or similar search criteria ...

Reply:
If this is just looking for a UIViewController class is that needed?